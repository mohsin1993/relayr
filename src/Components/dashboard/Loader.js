import React from 'react';
const Loader = () => <div className='loading'>loading...</div>;
export default Loader;
