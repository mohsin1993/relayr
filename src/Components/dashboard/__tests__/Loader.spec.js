import React from 'react';
import snapshot from 'check-snapshot';
import Loader from '../Loader';

describe('Loader', () => {
  it('It renders loader correctly', () => {
    snapshot(<Loader />);
  });
});
