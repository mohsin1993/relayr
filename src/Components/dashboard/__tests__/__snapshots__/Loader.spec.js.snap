// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader It renders loader correctly 1`] = `
<div
  className="loading"
>
  loading...
</div>
`;
